components/scream/src/share/io/output_manager.hpp

```diff
@@ -0,0 +1,179 @@
+#ifndef SCREAM_OUTPUT_MANAGER_HPP
+#define SCREAM_OUTPUT_MANAGER_HPP
+
+#include "share/io/scorpio_output.hpp"
+#include "share/io/scream_scorpio_interface.hpp"
+
+#include "ekat/mpi/ekat_comm.hpp"
+#include "ekat/ekat_parameter_list.hpp"
+#include "share/field/field_repository.hpp"
+#include "share/grid/grids_manager.hpp"
+
+namespace scream
+{
+
+class OutputManager
```

Comment:
Can you add some comments that describe this class, its purpose, and each of its methods? It's easiest by far to write this stuff down when one is working on it.